databricks/koalas/frame.py

```diff
@@ -6782,6 +6782,27 @@ def melt(self, id_vars=None, value_vars=None, var_name=None,
         0  a         B          1
         1  b         B          3
         2  c         B          5
+
+        >>> df.melt(id_vars='Z')  # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+          ...
+        KeyError: "The following 'id_vars' are not present in the DataFrame: [('Z',)]"
+
+        >>> df.melt(value_vars='Z')  # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+          ...
+        KeyError: "The following 'value_vars' are not present in the DataFrame: [('Z',)]"
+
+        >>> df.columns = pd.MultiIndex.from_tuples([('X', 'A'), ('X', 'B'), ('Y', 'C')])
+        >>> df.melt(id_vars=('A',))  # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+          ...
+        ValueError: id_vars must be a list of tuples when columns are a MultiIndex
+
+        >>> df.melt(value_vars=('A',))  # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+          ...
+        ValueError: value_vars must be a list of tuples when columns are a MultiIndex
```

Comment:
Could you move these to `DataFrameTest.test_melt`?

Reply:
Ok, Thanks for reviewing.